tensorcircuit/mpscircuit.py

```diff
@@ -396,21 +661,25 @@ def wavefunction(self, form: str = "default") -> Tensor:
 
     state = wavefunction
 
-    # TODO(@refraction-ray): mps form quvector
-
     def copy_without_tensor(self) -> "MPSCircuit":
         """
         Copy the current MPS without the tensors.
 
         :return: The constructed MPS
         :rtype: MPSCircuit
         """
+        from copy import copy
+
         result: "MPSCircuit" = MPSCircuit.__new__(MPSCircuit)
         info = vars(self)
         for key in vars(self):
             if key == "_mps":
                 continue
-            setattr(result, key, info[key])
+            if backend.is_tensor(info[key]):
+                copied_value = backend.copy(info[key])
+            else:
+                copied_value = copy(info[key])
+            setattr(result, key, copied_value)
         return result
 
     def copy(self) -> "MPSCircuit":
```

Comment:
not sure _mps.copy() support non-numpy array or not, suggest `backend.copy()`

Reply:
the same applies to `conj` method

Reply:
_mps.copy() is implemented in the FiniteMPS class in tensorcircuit code (not tn.FiniteMPS!), which is essentially doing backend.copy for tensors and a reconstruction of the FiniteMPS object, so should be fine?

Reply:
my bad, thought `_mps` is a numpy array before